src/components/core/breadcrumb/breadcrumb.test.tsx

```diff
@@ -0,0 +1,26 @@
+import { render, screen, within, fireEvent } from 'test-setup';
+import Breadcrumb from './breadcrumb';
+
+const items = ['Home', 'About', 'CFD', 'mt5'];
```

Comment:
MT5 in capatal case ?

Reply:
there was no test for this component and I just added some basic tests and used the data provided in storybook of the component. and it's suppose to be the current page in the breadcrumb and I guess it should be lowercase :sweat_smile: I think.